renv/cookies/posix/{{cookiecutter.dirname}}/.Renviron

```diff
@@ -1,3 +1,6 @@
 R_LIBS_USER={{cookiecutter.__R_LIBS_USER__}}
-R_HOME={{cookiecutter.__R_HOME__}}
-R_INCLUDE_DIR={{cookiecutter.__R_INCLUDE_DIR__}}
\ No newline at end of file
+R_LIBS_SITE={{cookiecutter.__R_LIBS_SITE__}}
+#R_HOME={{cookiecutter.__R_HOME__}}
```

Comment:
Why commented out here?

Reply:
```suggestion
```

Reply:
Looks like there were plenty of oversights.  I was doing lots of testing, and I commented this out to see if that was part of the problem.  Looks like I accidentally commited it this way.